tap_adroll/client.py

```diff
@@ -12,27 +12,37 @@
 
 class AdrollAuthenticationError(Exception):
     pass
-
-
 class AdrollClient():
-    def __init__(self, config_path, config):
-        token = {
-            'access_token': config['access_token'],
-            'refresh_token': config['refresh_token'],
-            'token_type': 'Bearer',
-            # Set expires_in to a negative number to force the client to reauthenticate
-            'expires_in': '-30'
-        }
-        extra = {
-            'client_id': config['client_id'],
-            'client_secret': config['client_secret']
-        }
+    def __init__(self, config_path, config, dev_mode = False):
+        self.dev_mode = dev_mode
         self.config_path = config_path
-        self.session = OAuth2Session(config['client_id'],
-                                     token=token,
-                                     auto_refresh_url=TOKEN_REFRESH_URL,
-                                     auto_refresh_kwargs=extra,
-                                     token_updater=self._write_config)
+        self.config = config
+        self.authenticate_request()
+
+    def authenticate_request(self):
+        if self.dev_mode :
+            self.session = requests.Session()
+            try :
+                self._access_token = self.config['access_token']
+            except KeyError as err:
+                raise Exception("Unable to locate access token in config") from err
```

Comment:
The intention of try-catch block is to check the `access_token` within config. Should we handle it using the `if` condition?

```
if 'access_token' not in self.config.keys():
     raise Exception("Unable to locate access token in config")
```

Reply:
made the changes